packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -168,6 +173,59 @@ focallength_px = prediction["focallength_px"]`;
 	return [installSnippet, inferenceSnippet];
 };
 
+export const derm_foundation = (model: ModelData): string[] => [
```

Comment:
oof! this is a bit too huge, is there anyway to reduce the size of this snippet? In general we try to keep the snippets as low as possible.